### PR TITLE
Fix: prevents onclick trigger for non truncated axis row label (fixes #90)

### LINKED ISSUE #90: onClick triggers popup for Axis Label Row labels which are not truncated
**Describe the bug**
onClick triggers popup for Axis Label Row items which are not truncated

**To Reproduce**

1. Go to https://engineering.cerner.com/carbon-graphs/#/bar/axis-info-text-labels
2. Click on 
image

3. Now click on 
image

4. Popup should not load

**Expected behavior**
onClick should be triggered only for truncated labels. 

**Screenshots**
image

**Desktop (please complete the following information):**

-   OS: [e.g. iOS] Windows 10
-   Browser [e.g. chrome, safari] Chrome

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by avinashg1994**
**What is the purpose of this pull request? (Check any that's applicable)**

-   [ ] Feature
-   [X] Bug fix
-   [ ] Dependency updates
-   [ ] Documentation updates
-   [ ] Build related changes
-   [ ] Changes an existing functionality
-   [ ] Other, please explain:

**Does this introduce a breaking change?**

-   [ ] Yes
-   [x] No

**What changes did you make? (Give an overview)**
Added a condition which allows attaching of an `onClick` event and setting of `aria-disabled` only when the `axisInfoRow` label is truncated.

**Add any screenshots**
When clicked on truncated label
Screenshot 2019-10-04 at 6 39 03 PM

When clicked on non-truncated label
Screenshot 2019-10-04 at 6 39 47 PM

### CODE REVIEW

File: src/main/js/controls/Bar/helpers/axisInfoRowHelpers.js [hunk 1]
```diff
@@ -58,17 +58,28 @@ const createAxisInfoRowLabel = (
...
-            );
-        });
+        .attr("aria-hidden", config.shownTargets.indexOf(uniqueKey) < 0);
+    if (
+        shouldTruncateLabel(
+            textLabelList.value.label.display,
+            textLabelList.value.characterCount
+        )
+    ) {
+        axisInfoRow
```
Comment (abhijit945) [inline src/main/js/controls/Bar/helpers/axisInfoRowHelpers.js]: +1 for Primary `display` but we need the same behavior with `secondaryDisplay` as well. 

Try adding a large text for **only** `secondaryDisplay` and see what happens with the above code enabled. Also check out `label` row here: https://github.com/cerner/carbon-graphs/blob/master/docs/controls/Bar.md#axis-info-row-value on how to update the values. I think there is an example in the site which you can play around with!

Comment (avinashg1994) [reply src/main/js/controls/Bar/helpers/axisInfoRowHelpers.js]: I agree, in the scenario when the value of only `secondaryDisplay` (not `display`) exceeds the characterCount, the onClick is disabled.
So, implemented the same behaviour for `secondaryDisplay` as well.

**Comment by abhijit945** [comment]
@avinashg1994 
Can you include some of your teammates for this review? Need one more +1 for this.

**Pull request merged**